Riot/Modules/SecretsRecovery/RecoverWithKey/SecretsRecoveryWithKeyViewController.swift

```diff
@@ -0,0 +1,279 @@
+/*
+ Copyright 2020 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import UIKit
+import MobileCoreServices
+
+final class SecretsRecoveryWithKeyViewController: UIViewController {
+    
+    // MARK: - Properties
+    
+    // MARK: Outlets
+    
+    @IBOutlet private weak var scrollView: UIScrollView!
+    
+    @IBOutlet private weak var shieldImageView: UIImageView!
+    
+    @IBOutlet private weak var informationLabel: UILabel!
+    
+    @IBOutlet private weak var recoveryKeyTitleLabel: UILabel!
+    @IBOutlet private weak var recoveryKeyTextField: UITextField!
+    @IBOutlet private weak var recoveryKeyTextFieldBackgroundView: UIView!
+    
+    @IBOutlet private weak var importFileButton: UIButton!
+    
+    @IBOutlet private weak var unknownRecoveryKeyButton: UIButton!
```

Comment:
I cannot understand the presence of this button here.
There is nothing we can do if the user lost their key, no?